ui/src/shared/reducers/flags.ts

```diff
@@ -28,10 +28,16 @@ export default (state = defaultState, action: Actions): FlagState => {
     case SET_FEATURE_FLAGS:
       // just setting the loading state
       if (!action.payload.flags) {
-        return {
+        const newState = {
           ...state,
           status: action.payload.status,
         }
+
+        if (!state.hasOwnProperty('original')) {
```

Comment:
this clears out old flags if none are returned?

Reply:
@drdelambre it makes sure the [flag state's structure](https://github.com/influxdata/influxdb/blob/master/ui/src/shared/reducers/flags.ts#L20) has the right shape and that `original` isn't `undefined`

it used to look like:

```
{status: 'loading', override: {}}
```
now it looks like:
```
{status: 'loading', override: {}, original: {}}
```

[`getFlags` dispatches `setFlags`](https://github.com/influxdata/influxdb/blob/master/ui/src/shared/actions/flags.ts#L50) with an empty second argument when it starts loading the flags. i noticed this while trying to track down this issue and changed it just to see if it fixed it. it didn't fix it, but i kept it in because otherwise it returns a state object that is missing a key.